src/cysignals/signals.pyx

```diff
@@ -193,22 +205,23 @@ def init_cysignals():
     import signal
     old = signal.signal(signal.SIGINT, python_check_interrupt)
 
-    setup_alt_stack()
+    IF UNAME_SYSNAME != 'Windows':
+        setup_alt_stack()
     setup_cysignals_handlers()
 
     # Set debug level to 2 by default (if debugging was enabled)
     _set_debug_level(2)
 
     return old
 
-
-def _setup_alt_stack():
-    """
-    This is needed after forking on OS X because ``fork()`` disables
-    the alt stack. It is not clear to me whether this is a bug or
-    feature...
-    """
-    setup_alt_stack()
+IF UNAME_SYSNAME != 'Windows':
```

Comment:
No need for that.

Reply:
`undefined reference to 'setup_alt_stack'` if removed